server/handlers/meshery_filter_handler.go

```diff
@@ -138,6 +139,10 @@ func (h *Handler) handleFilterPOST(
 
 		mesheryFilter := parsedBody.FilterData
 
+		// encode the filter binary data to base64 string before processing it further.
```

Comment:
Meshery Server and Remote Providers will decode upon request for download, correct?

Reply:
It should, although I've not verified it.